pkg/pods/pod_test.go

```diff
@@ -19,6 +19,8 @@ import (
 	"github.com/square/p2/pkg/types"
 	"github.com/square/p2/pkg/util"
 
+	"reflect"
```

Comment:
That instantaneous feeling of dread when I get to a file with a `+     "reflect"`